pygmo/docstrings.cpp

```diff
@@ -1938,4 +1938,154 @@ Runs the fast non dominated sorting algorithm on the input *points*
 )";
 }
 
+std::string hv_init1_docstring()
+{
+    return R"(__init__(pop)
+
+Constructor from population
+
+Args:
+    pop (:class:`~pygmo.core.population`): the input population
+
+Raises:
+    ValueError: if *pop* contains a single-objective or a constrained problem
+
+Examples:
+    >>> from pygmo import *
+    >>> pop = population(prob = zdt(id = 1), size = 20)
+    >>> hv = hypervolume(pop = pop)
+
+See also the docs of the C++ class :cpp:class:`pagmo::hypervolume`.
+
+)";
+}
+
+std::string hv_init2_docstring()
+{
+    return R"(__init__(points)
+
+Constructor from points
+
+Args:
+    points (2d-array like object): the points
+
+Raises:
+    ValueError: if *points* is inconsistent
+
+Examples:
+    >>> from pygmo import *
+    >>> points = [[1,2],[0.5, 3],[0.1,3.1]]
+    >>> hv = hypervolume(points = points)
+
+See also the docs of the C++ class :cpp:class:`pagmo::hypervolume`.
+
+)";
+}
+
+std::string hv_compute_docstring()
+{
+    return R"(hypervolume.compute(ref_point, hv_algo = auto)
+
+Computes the hypervolume with the supplied algorithm. If no algorithm
+is supplied,  then an exact hypervolume algorithm is automatically selected
+specific for the point dimension.
+
+Args:
+    ref_point (2d-array like object): the points
+    hv_algo (``pygmo.hv_algorithm``): hypervolume algorithm to be used
```

Comment:
Could you put here (and elsewhere, similarly) a hyperlink to ``pygmo.hv_algorithm`` instead?